server/services/stats/side/test.go

```diff
@@ -6,12 +6,14 @@ import (
 	"time"
 
 	"fmt"
+	"math/rand"
```

Comment:
![image](https://user-images.githubusercontent.com/975907/28197825-7222dfb6-686a-11e7-8779-a2363ba768fd.png)
are you sure it built before?